internal/handlers/common/parse_osrelease.go

```diff
@@ -34,7 +36,16 @@ func parseOSRelease(r io.Reader) (string, string, error) {
 			continue
 		}
 
-		configParams[str[0]] = str[1]
+		value := strings.Join(str[1:], "")
+		unquotedValue, err := strconv.Unquote(value)
+
+		if errors.Is(err, nil) {
+			configParams[str[0]] = unquotedValue
+		} else if errors.Is(err, strconv.ErrSyntax) {
+			configParams[str[0]] = value
+		} else {
+			return "", "", lazyerrors.Error(err)
+		}
```

Comment:
That part will look more clear with `switch`, I think.
```suggestion
		switch {
		case errors.Is(err, nil):
			configParams[str[0]] = unquotedValue
		case errors.Is(err, strconv.ErrSyntax):
			configParams[str[0]] = value
		default:
			return "", "", lazyerrors.Error(err)
		}
```